packages/components/tests/vue/components/morph-parts/principal-parts.test.js

```diff
@@ -226,6 +226,6 @@ describe('principal-parts.test.js', () => {
     })
 
     expect(cmp.find('.alpheios-principal-parts__lemma_index').text()).toEqual('2')
-    expect(cmp.find('.alpheios-principal-parts__groupitem').text().replace(/(\r\n|\n|\r)/gm, '').replace(/  +/g, ' ')).toEqual('Cupido Cupidinis')
+    expect(cmp.find('.alpheios-principal-parts__groupitem').text().replace(/(\r\n|\n|\r)/gm, '').replace(/  +/g, ' ')).toEqual('CupidoCupidinis')
```

Comment:
I think this is why we needed the space in the vue component

Reply:
I've noticed that in the popup the words are shown with spaces between the commas:
![image](https://user-images.githubusercontent.com/18631055/79897133-50553900-841a-11ea-8ac2-da71df728e51.png)
A removal of spaces in a Vue component fixed that. But here we have no commas, just spaces. That's why words a glued together. But this is not how they are shown in a popup, at least not in my tests.

If we need to keep those spaces, I can return them back. If not, I can think of how to change the coding of this test maybe.

Reply:
I could be wrong, and so maybe let's leave it as you have it and see what happens.